src/rtcModule/webrtc.cpp

```diff
@@ -2359,9 +2392,24 @@ void Call::onClientLeftCall(Id userid, uint32_t clientid)
         {
             mManager.launchCallRetry(mChat.chatId(), sentAv(), false);
         }
+    }
```

Comment:
Maybe in the block above we should preserve the `cancelSessionRetryTimer()` and the `destroyIfNoSessionsOrRetries()`?? Not sure, but worths to check it.

Reply:
That part of code was added by merge. It's not necessary